src/Generator.Bind/LicenseResource.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Reflection;
+
+namespace Bind
+{
+    /// <summary>
+    /// Class for handling embedded license text.
+    /// </summary>
+    internal class LicenseResource
+    {
+        private static string _license;
+
+        /// <summary>
+        /// Gets and returns the license text from the embedded resource.
+        /// </summary>
+        public static string LicenseText
+        {
+            get
+            {
+                if (_license != null)
+                {
+                    return _license;
+                }
+
+                var streamReader = new StreamReader(Assembly.GetAssembly(typeof(Program)).GetManifestResourceStream("Specifications/License.txt"));
+                _license = streamReader.ReadToEnd();
+                streamReader.Dispose();
+                return _license;
+            }
+        }
+    }
+}
```

Comment:
End the file with a newline - your IDE should complain about this, but I might have slipped up in the rulesets and not enabled that.

Reply:
Nope, StyleCop gave me an error so i added the newline then the build went through without errors. Using rider as IDE, maybe some issues with rider/git resulting in removing trailing newlines while pushing?

Reply:
Jesus how the hell do you even spot tiny things like that 😂

Reply:
It's an IDE setting, I think - you can turn of stripping of newlines at the end of the file. I use Rider myself, and I think I saw that somewhere.

@DylanFPS I have special eyes :eyes:

Reply:
Even if I open the file in another editor, the newline is there. But i looked at some files of your actual OpenAL merge, the newlines are in the files, but it seems GitHub doesn't display trailing newlines.